src/condition.h

```diff
@@ -46,7 +46,7 @@ const std::unordered_set<std::string> complex_conds = { {
         "days_since_cataclysm", "is_season", "mission_goal", "u_has_var", "npc_has_var", "expects_vars",
         "u_has_skill", "npc_has_skill", "u_know_recipe", "u_compare_var", "npc_compare_var",
         "u_compare_time_since_var", "npc_compare_time_since_var", "is_weather", "mod_is_loaded", "one_in_chance", "x_in_y_chance",
-        "u_is_height", "npc_is_height",
+        "u_is_height", "npc_is_height", "math",
```

Comment:
It lets dynamic lines accept math statements for their conditions. I assume it was not enabled when the whole math logic was added, since there were no use cases, but it worked without needing any further modifications.